src/project/FileTreeView.js

```diff
@@ -478,7 +478,51 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
+              case 'htm':
+              case 'htx':
+              case 'md':
+              case 'markdown':
+              case 'html':
+              case 'xml':
+              case 'xhtml':
+                fileType = "html";
+                break;
+              case 'ico':
+              case 'bmp':
+              case 'svg':
+              case 'png':
+              case 'ico':
+              case 'jpg':
+              case 'jpe':
+              case 'jpeg':
+              case 'gif':
+                fileType = "image";
+                break;
+              case 'css':
+              case 'less':
+                fileType = "css";
+                break;
+              case 'js':
+              case 'jsx':
+              case 'json':
+                fileType = "js";
+                break;
+              default:
+              var isBinary = LanguageManager.getLanguageForPath(fullname).isBinary();
```

Comment:
Indent this part of the code, like you do with other code under a `case...:` above.

Also, let's simplify this to one line:

```js
default:
    fileType = LanguageManager.getLanguageForPath(fullname).isBinary() ? "binary" : "default";
    break;
```

Also, you're mixing single and double quotes in this file, which is not generally a good idea.  You should stick with the default used in the rest of the file, which seems to be double (you use single and double in this code).